static/js/csrf-fix.js

```diff
@@ -0,0 +1,10 @@
+$.ajaxPrefilter(function(options, originalOptions, jqXHR) {
+	var token;
```

Comment:
Should we alter this filter to exclude crossdomain requests? 

As it is this has the possiblility to leak CSRF token to external hosts via ajax.

we would likely be able to prevent this by adding `if (options.crossDomain) { return; }`